### ISSUE TITLE
Shame this doesn't work

### ISSUE THREAD

**Issue by AaronLayton** [mauriciopoppe/PojoViz#7]
I expect you lost interest in this project a while ago though, anyone visiting here know of an alternative?

**Comment by mauriciopoppe** [comment]
yep, even though the library still works the webpage uses what's now a version of webcomponents that's deprecated and unfortunately, I've moved on and cannot work on this anymore

I'd suggest using http://www.objectplayground.com/ instead

**Comment by AaronLayton** [comment]
Brill - thanks for taking the time to reply @mauriciopoppe
